automation/README.md

```diff
@@ -55,8 +47,8 @@ You may change the test type by using:
  * `./automation/run-tests.sh --test-type unit_py37`:
    Unit tests in Python 3.7 using 'nmstate/fedora-nmstate-dev' docker image.
 
- * `./automation/run-tests.sh --test-type integ --el7`:
-   Integration tests using 'nmstate/centos7-nmstate-dev' docker image.
+ * `./automation/run-tests.sh --test-type integ --el`:
```

Comment:
What happened to the `8`?

Reply:
@cathay4t changed it like this in the other PR, as long as we are not supporting multiple enterprise Linuxes, the version does not make sense IMHO.

Reply:
changed it to `--el8`.